public/docs/_examples/toh-6/dart/web/main.dart

```diff
@@ -0,0 +1,24 @@
+// #docplaster
+// #docregion final
+// #docregion v1
+import 'package:angular2/core.dart';
+import 'package:angular2/platform/browser.dart';
+import 'package:http/http.dart';
+import 'package:angular2_tour_of_heroes/app_component.dart';
+// #enddocregion v1
+import 'package:angular2_tour_of_heroes/in_memory_data_service.dart';
+
```

Comment:
As agreed, I've changed the providers list argument to `bootstrap()` to conform to the workaround mentioned in https://github.com/angular/angular/issues/9673, namely:
```dart
  bootstrap(AppComponent, [
    provide(BrowserClient, useFactory: () => new BrowserClient(), deps: [])
  ]);
```